web-app/helpers/certificate-designs.ts

```diff
@@ -31,23 +27,36 @@ function addText(canvas: Canvas, text: string, font: string, fillStyle: string,
 export async function populateDeveloperCert(canvas: Canvas, details: any) {
   // TODO
   console.log('populateDeveloperCert', { details });
-  const { tokenId, date, programName, accountName, expiration } = details;
+  const { tokenId, date, programName, accountName, expiration, programDescription, instructor } = details;
 
-  const fillStyle = `#${Math.floor(Math.random() * 16777215).toString(16)}`; // This is just temporary and will be removed once we have final designs for each program's certificate. https://css-tricks.com/snippets/javascript/random-hex-color/
-  const font = `40px '${fontFamily}' bold`;
+  const newDate = date.split(' ')[0];
+  const newExpiration = expiration.split(' ')[0];
+  const programDescription1 = programDescription.split(',')[0];
+  const programDescription2 = programDescription.split(',')[1];
```

Comment:
I tried to explain why i did such a thing in here: https://discord.com/channels/828768337978195969/906115083250307103/941290874573320203

But basically, if I don't split the description, it looks like this
![image](https://user-images.githubusercontent.com/45902312/153601833-ce07d249-c976-4e94-8d38-8fdaeb0b7f5b.png)

After splitting it,
![image](https://user-images.githubusercontent.com/45902312/153601926-8d3243f5-33b8-4638-9ea7-d0369bb9a305.png)

Maybe it is not a best practice but idk how to fix it otherwise. 
So for now, I can not change it.